components/style/style_adjuster.rs

```diff
@@ -6,7 +6,10 @@
 //! for it to adhere to the CSS spec.
 
 use app_units::Au;
-use properties::{self, ComputedValues, StyleBuilder};
+use properties::{self, CascadeFlags, ComputedValues};
+use properties::{SKIP_ROOT_AND_ITEM_BASED_DISPLAY_FIXUP, StyleBuilder};
+#[cfg(feature = "gecko")]
```

Comment:
Oh yeah, that's a better idea.  (Reluctant to give up the 42 minutes of progress the test run has had, though!)